packages/itwinui-css/src/select/select.scss

```diff
@@ -35,7 +35,7 @@
   }
 
   &:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true']) {
-    --_iui-list-item-icon-fill: var(--iui-color-icon-disabled);
+    --_iui-list-item-icon-fill: var(--_iui-field-color-icon-disabled);
```

Comment:
It's weird that we are setting disabled fill color, but not setting default / hover state fill color within `select.scss`.

Reply:
It seems like some of the icon fill logic is currently being handled by this [menu mixin](https://github.com/iTwin/iTwinUI/blob/dac60976090fb86069bb0a4807a0ecd0a550e1fa/packages/itwinui-css/src/menu/mixins.scss#L181-L189). Should probably also move this logic into field as well.

Reply:
Removed the menu mixin.